package.json

```diff
@@ -49,6 +49,7 @@
     "@cfpb/cfpb-expandables": "^0.35.0",
     "@cfpb/cfpb-forms": "^0.35.0",
     "@cfpb/cfpb-icons": "^1.2.0",
+    "@fontsource/source-sans-3": "^5.2.8",
```

Comment:
Ideally, you're not installing this, and are installing the DS v4.x version, but I see that this project is still using `.less`(!), so that would be a greater lift to go to DS 4.